ovos_dinkum_listener/service.py

```diff
@@ -681,7 +682,36 @@ def _save_stt(self, audio_bytes, stt_meta, save_path=None):
             stt_audio_dir = Path(f"{self.default_save_path}/utterances")
         stt_audio_dir.mkdir(parents=True, exist_ok=True)
 
-        filename = hash_sentence(stt_meta["transcription"])
+        listener = self.config.get("listener")
```

Comment:
theoretically this could be None, ` listener = self.config.get("listener", {})`

Reply:
in practice it will never be missing, but overall let's try to handle that use case, as the default config file can be replaced via os env vars. but one could argue that would be user error if sections are missing....